src/api/html5media.js

```diff
@@ -313,10 +313,12 @@
         replacement.style.cssText = element.style.cssText;
         replacement.className = element.className;
         replacement.title = element.title;
-        replacement.style.display = "block";
+        if (!element.style.display) {
+            replacement.style.display = "block";
+        }
```

Comment:
A thought - could we copy all the style properties over? So the logic would be:

1. `replacement.style = element.style`.
2. For each "default" html5media style, set that style on the element if it's not already set.